Misc changes for image-to-image (#329)
huggingface/huggingface.js: packages/tasks/src/image-to-image/about.md

### Style transfer

> One of the most popular use cases of image-to-image is style transfer. Style transfer models can convert regular photography into a painting in the style of a famous painter.

Reviewer: ```suggestion
One of the most popular use cases of image-to-image is style transfer. Style transfer models can convert a normal photography into a painting in the style of a famous painter.
```

(I'd keep the `a` article here)

## Task Variants

### Image inpainting

Image inpainting is widely used during photography editing to remove unwanted objects, such as poles, wires, or sensor
dust.

### Image colorization

> Old, black, and white images can be brought up to life using an image colorization model.

Reviewer: ```suggestion
Old or black and white images can be brought up to life using an image colorization model.
```

## ControlNet

> Controlling the outputs of diffusion models only with a text prompt is a challenging problem. ControlNet is a neural network type that provides an image-based control to diffusion models. These controls can be edges or landmarks in an image.

Reviewer: ```suggestion
Controlling the outputs of diffusion models only with a text prompt is a challenging problem. ControlNet is a neural network model that provides image-based control to diffusion models. Control images can be edges or other landmarks extracted from a source image.
```

Many ControlNet models were trained in our community event, JAX Diffusers sprint. You can see the full list of the ControlNet models available [here](https://huggingface.co/spaces/jax-diffusers-event/leaderboard).

## Most Used Model for the Task

Pix2Pix is a popular model used for image-to-image translation tasks. It is based on a conditional-GAN (generative adversarial network) where instead of a noise vector a 2D image is given as input. More information about Pix2Pix can be retrieved from this [link](https://phillipi.github.io/pix2pix/) where the associated paper and the GitHub repository can be found.

> The below images show some of the examples shared in the paper that can be obtained using Pix2Pix. There are various cases this model can be applied on. It is capable of relatively simpler things, e.g., converting a grayscale image to its colored version. But more importantly, it can generate realistic pictures from rough sketches (can be seen in the purse example) or from painting-like images (can be seen in the street and facade examples below).

Reviewer: ```suggestion
The images below show some examples extracted from the Pix2Pix paper. This model can be applied to various use cases. It is capable of relatively simpler things, e.g., converting a grayscale image to its colored version. But more importantly, it can generate realistic pictures from rough sketches (can be seen in the purse example) or from painting-like images (can be seen in the street and facade examples below).
```
